README.md

```diff
@@ -105,3 +105,11 @@ Additional information about supported hardware can be found in [PX4 user Guide
 ## Project Roadmap
 
 A high level project roadmap is available [here](https://www.dronecode.org/roadmap/).
+
+## Governance
+
+The PX4 flight stack is hosted under the governance of the [Dronecode Project](https://www.dronecode.org/).
```

Comment:
I think that needs further clarification. Dronecode has no say in the technical direction of the project per se (as the Linux foundation has no say on the Linux Kernel), and this comes across differently.

Reply:
Agreed, I think I should take a step back here and explain where I'm coming from.

Dronecode has no say in the technical direction, and neither any of its members, that much should be clear, the project remains with the same domain leadership and transparent governance structure. My intention was to show the relationship between Dronecode and PX4. I realize now that this requires far more than a few words to explain, I'm going to close this PR and come back with a topic on the next dev call instead, I definitely jumped a few steps here.

Reply:
Remember we have a bunch of these now scattered around our other docs, so I am assuming we can live with this in the meantime :-). I think that perhaps the word to avoid here is "governance". 

The word I was looking for that closest matches the relationship is *auspices* (protection/support). There are other synonyms [here](https://www.thesaurus.com/browse/auspices) - see patronage, sponsorship etc.